package.json

```diff
@@ -176,6 +175,9 @@
     "*pkg/**/*.go": [
       "gofmt -w -s",
       "git add"
+    ],
+    "*.{ts,tsx}": [
```

Comment:
This check happens in the hook itself. See implementation for details.

Reply:
Oh, I see.

Hm, it's still a bit slow (at least on my machine) comparing with the proposed implementation. I guess that's because `ts-node` is slow by itself.

But if we are fine with that, I will leave only the repetitions removal

Reply:
Sorry @markelog for not responding on slack regarding the slowness here, I did also on my old mac notice that this was indeed a bit slow. But not sure this is a solution either as it sidesteps all pre-commit hooks for other files and does not do the file location filtering (that applies the right precommit action depending on file and location) that @dprokop added in his recent PR so we would have to find another solution for that as merging this would loose that feature.

Reply:
I see! Will put some more thought to it and consult with the team beforehand next time :)